src/handlers/dns/DnsStreamHandler.cpp

```diff
@@ -309,6 +313,30 @@ bool DnsStreamHandler::_filtering(DnsLayer &payload, [[maybe_unused]] PacketDire
     return true;
 }
 
+const std::bitset<64> DnsStreamHandler::_process_dns_groups(const Configurable *metrics_config)
+{
+    // default enabled groups
+    std::bitset<64> groups;
+    groups.set(group::DnsMetrics::Counters);
+    groups.set(group::DnsMetrics::TopDnsWire);
+    groups.set(group::DnsMetrics::DnsTransactions);
+    groups.set(group::DnsMetrics::TopQnames);
+
+    if (metrics_config->config_exists("enable")) {
+        for (const auto &group : metrics_config->config_get<StringList>("enable")) {
+            groups.set(_group_metrics.at(group));
```

Comment:
this `.at()` call we'll be processing arbitrary user input, so we need to catch when it doesn't match a key and ignore/warn

Reply:
It is being called in the start, where we also throw other things. So, you think we should simple handle this here and not throw to upstream?

Reply:
well I am mostly worried that the built in exception isn't useful enough and we want a custom error message